src/lib.rs

```diff
@@ -11,19 +11,95 @@
 //! On the write side, `async-bincode` buffers the serialized values, and asynchronously sends the
 //! resulting bytestream.
 #![deny(missing_docs)]
+#[cfg(not(any(feature = "futures", feature = "tokio")))]
+compile_error!("async-bincode: Enable at least one of \"futures\" or \"tokio\".");
 
+#[macro_use]
 mod reader;
+#[macro_use]
 mod stream;
+#[macro_use]
 mod writer;
 
-pub use crate::reader::AsyncBincodeReader;
-pub use crate::stream::AsyncBincodeStream;
-pub use crate::writer::AsyncBincodeWriter;
```

Comment:
To avoid making this a backwards-incompatible change, I'd like to re-export the types from `mod tokio` here and do a point release with that in place.

Reply:
Sure, easy to do.